src/pykeen/losses.py

```diff
@@ -201,15 +317,10 @@ def forward(
 }
 
 #: HPO Defaults for losses
-losses_hpo_defaults: Mapping[Type[Loss], Mapping[str, Any]] = {
-    MarginRankingLoss: dict(
-        margin=dict(type=int, low=0, high=3, q=1),
-    ),
-}
-# Add empty dictionaries as defaults for all remaining losses
-for cls in _LOSSES:
-    if cls not in losses_hpo_defaults:
-        losses_hpo_defaults[cls] = {}
+losses_hpo_defaults: Mapping[Type[Loss], Mapping[str, Any]] = defaultdict(dict)
+losses_hpo_defaults[MarginRankingLoss] = dict(
```

Comment:
after you're done with your part of the PR, I will reorganize this such that each loss function lists its own defaults

Reply:
Done in b2f619e